ee/hogai/videos/session_moments.py

```diff
@@ -158,6 +158,8 @@ async def _generate_video_for_single_moment(
                     "playback_speed": SHORT_VALIDATION_VIDEO_PLAYBACK_SPEED,
                     # Keeping default values
                     "mode": "screenshot",
+                    # Display additional metadata for LLMs in the video
+                    "show_llm_metadata": True,
```

Comment:
Flyby nit: `llm_metadata` is a bad name, this is just generic metadata that can be used for anything